autogen/json2class.cpp

```diff
@@ -1671,18 +1671,18 @@ void file_python_namespace(
    cpp << "   // create the " << nsname << " submodule\n";
    cpp << "   python::module submodule = module.def_submodule(\n";
    cpp << "      \"" << nsname << "\",\n";
-   cpp << "      \"GNDS v1.9 " << nsname << "\"\n";
+   cpp << "      \"GNDS " << Version << " " << nsname << "\"\n";
```

Comment:
I can't believe I missed that accidental hardcoding. :-/

Reply:
I just noticed that by chance.